requirements.txt

```diff
@@ -0,0 +1,2 @@
+cwlref-runner==1.0
+cwltool==1.0.20190831161204
```

Comment:
Could you add a comment explaining why we need both, if we do? We're just using the one commandline tool, I think, so what is the other one for?

Reply:
The runner is the CLI runner for the tool.